PSDepend/PSDepend.psd1

```diff
@@ -58,7 +58,7 @@ FormatsToProcess = 'PSDepend.Format.ps1xml'
 # NestedModules = @()
 
 # Functions to export from this module
-FunctionsToExport = '*'
+FunctionsToExport = @('Get-Dependency','Get-PSDependScript','Get-PSDependType','Import-Dependency','Install-Dependency','Invoke-DependencyScript','Invoke-PSDepend','Test-Dependency')
 
 # Cmdlets to export from this module
 CmdletsToExport = '*'
```

Comment:
Build is a bit... janky (i.e. changes to psd1 after testing, not pushing changes back to source).  Happy to move this to an explicit array, but it will get there during the build as is, [via this](https://github.com/RamblingCookieMonster/PSDepend/blob/master/psake.ps1#L78))

Reply:
@RamblingCookieMonster Ah! I should have known better than to think you'd leave it as a wild card 😉

Marking this resolved